### PR TITLE
doc_update/issues_template

### PR DESCRIPTION
**Pull request by uginf0urth**
added issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (0xAleksaOpacic) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Not sure how is this relevant?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (0xAleksaOpacic) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Not sure how is this relevant?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (0xAleksaOpacic) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: This is applicable for frontend not for hardhat script?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (0xAleksaOpacic) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: What if it's not, just a normal request?

**Review by 0xAleksaOpacic — CHANGES_REQUESTED** [review]
LGTM, i think we need to polish these a bit before merging. Left few minot comments

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (uginf0urth) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: hm. I deleted this from template O_o 
I will check it again

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (uginf0urth) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Sure. It is not relevant. I cleaned it from template, but... 🤷‍♂️

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (uginf0urth) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: Just not describe it here =) create it like a feature request

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (0xAleksaOpacic) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: Yes but template should match how we use the repository. Lets say someone wants to report bug, there is not go to -> click -> scroll 

In hardhat we only have running tests or tasks or set of tasks one by one

Comment (0xAleksaOpacic) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: Templates are not general for every repository, right?

**Review by 0xAleksaOpacic — APPROVED** [review]
